From 08b3e5e23bf4e410d59fb6de5914ecd918aba8c4 Mon Sep 17 00:00:00 2001
From: Larry Finger <Larry.Finger@lwfinger.net>
Date: Thu, 20 Mar 2014 02:08:30 -0500
Subject: [PATCH 392/393] Remove unused rtl8192d_PHY_ConfigRFWithParaFile, and
 fix so that scanning works with NetworkManager

There apparently was confusion between rtl8192d_PHY_ConfigRFWithHeaderFile and
rtl8192d_PHY_ConfigRFWithParaFile. The former is not an empty routine, but
the patch that said it was removing the latter actually substituted for the
former. This error likely happened when the patches for master were translated
to the kernel-version branch.

Signed-off-by: Larry Finger <Larry.Finger@lwfinger.net>
---
 hal/rtl8192d_phycfg.c    | 27 ---------------------------
 include/Hal8192DPhyCfg.h | 19 ++++++++++++++++---
 include/autoconf.h       |  2 +-
 3 files changed, 17 insertions(+), 31 deletions(-)

--- a/drivers/staging/rtl8192du/hal/rtl8192d_phycfg.c
+++ b/drivers/staging/rtl8192du/hal/rtl8192d_phycfg.c
@@ -1084,33 +1084,6 @@ PHY_RFConfig8192D(
 	return rtStatus;
 }
 
-/*-----------------------------------------------------------------------------
- * Function:    PHY_ConfigRFWithParaFile()
- *
- * Overview:    This function read RF parameters from general file format, and do RF 3-wire
- *
- * Input:	struct rtw_adapter *			adapter
- *			ps1Byte					pFileName
- *			enum RF_RADIO_PATH_E	eRFPath
- *
- * Output:      NONE
- *
- * Return:      RT_STATUS_SUCCESS: configuration file exist
- *
- * Note:		Delay may be required for RF configuration
- *---------------------------------------------------------------------------*/
-int
-rtl8192d_PHY_ConfigRFWithParaFile(
-	struct rtw_adapter *			adapter,
-	u8*				pFileName,
-	enum RF_RADIO_PATH_E		eRFPath
-)
-{
-	int	rtStatus = _SUCCESS;
-
-	return rtStatus;
-}
-
 /*  */
 /*-----------------------------------------------------------------------------
  * Function:    PHY_ConfigRFWithHeaderFile()
--- a/drivers/staging/rtl8192du/include/Hal8192DPhyCfg.h
+++ b/drivers/staging/rtl8192du/include/Hal8192DPhyCfg.h
@@ -272,9 +272,22 @@ int	PHY_MACConfig8192D(	struct rtw_adapt
 int	PHY_BBConfig8192D(	struct rtw_adapter *	adapter	);
 int	PHY_RFConfig8192D(	struct rtw_adapter *	adapter	);
 /* RF config */
-int	rtl8192d_PHY_ConfigRFWithParaFile(	struct rtw_adapter *	adapter,
-						u8 *pFileName,
-						enum RF_RADIO_PATH_E	eRFPath);
+/*-----------------------------------------------------------------------------
+ * Function:    PHY_ConfigRFWithParaFile()
+ *
+ * Overview:    This function read RF parameters from general file format, and do RF 3-wire
+ *
+ * Input:	struct rtw_adapter *			adapter
+ *			ps1Byte					pFileName
+ *			enum RF_RADIO_PATH_E	eRFPath
+ *
+ * Output:      NONE
+ *
+ * Return:      RT_STATUS_SUCCESS: configuration file exist
+ *
+ * Note:		Delay may be required for RF configuration
+ *---------------------------------------------------------------------------*/
+
 int	rtl8192d_PHY_ConfigRFWithHeaderFile(	struct rtw_adapter *			adapter,
 						enum RF_CONTENT			Content,
 						enum RF_RADIO_PATH_E	eRFPath);
--- a/drivers/staging/rtl8192du/include/autoconf.h
+++ b/drivers/staging/rtl8192du/include/autoconf.h
@@ -23,7 +23,7 @@
 
 //#define CONFIG_WAKE_ON_WLAN
 
-//#define CONFIG_92D_AP_MODE 1
+#define CONFIG_92D_AP_MODE 1
 
 //#define CONFIG_P2P	1
 
